mx.truffleruby/suite.py

```diff
@@ -81,7 +86,9 @@
             },
             "sha1": "3ba3cb50e9a0b87b645cd412a8f7b7296c314aa1",
             "sourceSha1": "2fdcca5bd2c629c8c120d46cc1ed325c994ea3eb",
-            "licence": "MIT"
+            "licence": [
```

Comment:
I noticed some of these are `licence` and some are `license`

Reply:
`mx` itself also uses both :(

Reply:
Fixed.